src/Traits/Subsection.php

```diff
@@ -111,7 +111,7 @@ protected static function get_link_text( $link_text ) {
 	 * @return string
 	 */
 	protected static function get_description( $description ) {
-		return $description[ self::get_locale() ] ?? $description['en'] ?? '';
+		return $description['text'][ self::get_locale() ] ?? $description['text']['en'] ?? '';
```

Comment:
The addon description does not currently have the text element in it. Is that something you suggest changing in the current models as well? For example see our current [Klarna Payments](https://github.com/krokedil/krokedil-settings-page-configs/blob/main/configs/klarna-payments-for-woocommerce.json#L186-L189) and [Avarda](https://github.com/krokedil/krokedil-settings-page-configs/blob/main/configs/klarna-payments-for-woocommerce.json#L186-L189) configs. I would not be opposed to that if we want to add something else to it, but i think we should then make sure it works like it does today still if that is not set yet.